janus_core/cli/geomopt.py

```diff
@@ -194,11 +184,16 @@ def geomopt(
     symmetry_tolerance
         Atom displacement tolerance for spglib symmetry determination, in Å.
         Default is 0.001.
+    file_prefix
+        Prefix for output filenames. Default is inferred from structure.
     out
         Path to save optimized structure, or last structure if optimization did not
         converge. Default is inferred from name of structure file.
-    traj
-        Path if saving optimization frames. Default is None.
+    write_traj
+        Whether to save a trajectory file of optimization frames.
+    traj_kwargs
+        Keyword arguments to pass to ase.io.write when saving trajectory.
+        If "filename" is not included, it is inferred from --file-prefix.
```

Comment:
```suggestion
        If "filename" is not included, it is inferred from `file_prefix`.
```
I normally just refer to the Python variables